### ISSUE TITLE
Construct join/invite pages for projects

### ISSUE THREAD

**Issue by tcmitchell** [GENI-NSF/geni-portal#19]
Start with the join/invite pages for projects. From there, we'll get feedback and iterate before rolling out to slices.

 _Imported from trac ticket [#19](http://trac.gpolab.bbn.com/proto-ch/ticket/19),  created by tmitchel on 05-23-2012 at 13:46, last modified: 05-30-2012 at 11:32_

**Comment by ahelsing** [comment]
Marshall will build the back-end request infrastructure

_Trac comment by ahelsing on 05-23-2012 at 13:56_

**Comment by ahelsing** [comment]
Project pages are checked in.

Marshall will tie these into the back end he is building.

Question: Do the portal pages email the project lead? Or does the PA do that?


_Trac comment by ahelsing on 05-29-2012 at 14:12_

**Comment by MarshallBrinn** [comment]
I believe this is done. There were a number of places to change and I think I went through all the paths but I certainly could have missed one. One thing I added was checks in the PA to make sure you're not already a member of the project before adding you. 

The portal pages send the email, not the PA. I'm not that comfortable with the PA getting involved in explicit workflows like this: it seems very tool specific.

_Trac comment by mbrinn (github user: MarshallBrinn) on 05-30-2012 at 11:29_
